pegen/pegen.c

```diff
@@ -2,6 +2,62 @@
 #include "pegen.h"
 #include "v38tokenizer.h"
 
+int
+raise_syntax_error(Parser *p, const char *errmsg, ...)
+{
+    // TODO: comvert from bytes offset to character offset
```

Comment:
Is it very onerous to just do this, rather than leaving a TODO?

Reply:
No, but I was not sure I understood what field was the comment referring to and I wanted to discuss this. What changes does this `TODO` require?

Reply:
I assume it's about column numbers being measured in bytes vs. characters, if the file has an encoding? You may have to attempt to repro this by creating a file or string with a unicode identifier followed by a syntax error to see what offset is displayed.